test/0.8.25/vaults/contracts/StakingVault__HarnessForTestUpgrade.sol

```diff
@@ -12,16 +12,18 @@ import {VaultHub} from "contracts/0.8.25/vaults/VaultHub.sol";
 import {IReportReceiver} from "contracts/0.8.25/vaults/interfaces/IReportReceiver.sol";
 import {IStakingVault} from "contracts/0.8.25/vaults/interfaces/IStakingVault.sol";
 import {IBeaconProxy} from "contracts/0.8.25/vaults/interfaces/IBeaconProxy.sol";
-import {VaultBeaconChainDepositor} from "contracts/0.8.25/vaults/VaultBeaconChainDepositor.sol";
+import {BeaconChainDepositLogistics} from "contracts/0.8.25/vaults/BeaconChainDepositLogistics.sol";
 
-contract StakingVault__HarnessForTestUpgrade is IBeaconProxy, VaultBeaconChainDepositor, OwnableUpgradeable {
+contract StakingVault__HarnessForTestUpgrade is IBeaconProxy, BeaconChainDepositLogistics, OwnableUpgradeable {
```

Comment:
It looks like it does not harness StakingValut but instead mocks the functionality. Rename to __Mock?